src/Resources/snippet/en_US/storefront.de-US.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
Probably typo in file name, should be en-US?

Reply:
Good catch. Should be en-US, of course.